packages/core/src/index.ts

```diff
@@ -35,14 +40,16 @@ const API: OnboardAPI = {
       updateNotify,
       customNotification,
       updateBalances,
-      updateAccountCenter
+      updateAccountCenter,
+      setPrimaryWallet
     }
   }
 }
 
+export type OnboardAPI = typeof API
```

Comment:
@aaronbarnardsound I like this update! Should we use this type in React/Vue packages instead of the OnboardAPI type in the types folder?

Reply:
That type is exported from this file and then imported by the React and Vue packages, so it already is!

Reply:
Ah I see, perfect! 🤘🏼